style.css

```diff
@@ -534,7 +534,6 @@ a {
 }
 
 @media screen and (min-width: 768px) {
-
   .header {
```

Comment:
As requested by the previous reviewer, kindly make sure that your design matches that of the provided design in terms of dimensions and margins since it is part of the project requirements :pray: 

Your design 
![your design](https://user-images.githubusercontent.com/65233860/204769998-d3a90fed-393b-486e-8d4a-4c329d9b6657.png)


The given design

![the design](https://user-images.githubusercontent.com/65233860/204770031-f30b2fba-4034-4507-aafe-eb44a1f61ed4.png)

Reply:
Done